core/plugins/org.polarsys.capella.core.sirius.ui/src/org/polarsys/capella/core/sirius/ui/listener/DAnnotationPrecommitListener.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2021 THALES GLOBAL SERVICES. All rights reserved. 
```

Comment:
can you use EPL 2 template like below in DAnnotationChangeTest.java

Reply:
Done